src/server.c

```diff
@@ -416,7 +344,14 @@ main(int argc, char **argv) {
     lwsl_notice("tty configuration:\n");
     if (server->credential != NULL)
         lwsl_notice("  credential: %s\n", server->credential);
-    lwsl_notice("  start command: %s\n", server->command);
+    
+    lwsl_notice("  start command: \n");
+    for (c = 0; c < argc; c++) {
+        lwsl_notice(" %s\n", argv[c]);
```

Comment:
Note that in my experience having each argument well marked and different is very usefull
if a nasty space is miss interpreted you can look for for the diff between `arg1` and `arg1 ` in a middle of a command

like
`fancy_pro arg1  arg2 arg3`
`fancy_pro arg1 arg2  arg3`

where both work differently because one is actually a spacing error, the other is actually a space in the arg2